### DISCUSSION TITLE
What do you think about my Clarity theme?

### DISCUSSION THREAD

**Discussion by zenblom** [datenstrom/community#739] [Ask a question] [answered]
Found this nice project a few days ago and really like how small but modularized it is with extensions!

However I'm missing a good theme for blogging so I made a quick port of the [Hugo](https://github.com/chipzoller/hugo-clarity) Clarity theme which is quite popular. It is not yet 1:1 on feature parity but most of it should be there and working as in Hugo. Never ever written a single line of php before so I'm sure there are improvements to be made but from my initial testing at least it seems stable so I thought I could share it: [Yellow Clarity theme](https://github.com/zenblom/yellow-clarity)

<p align="center"><img src="https://raw.githubusercontent.com/zenblom/yellow-clarity/main/clarity-screenshot1.png" width="795" alt="Screenshot1"></p>
<p align="center"><img src="https://raw.githubusercontent.com/zenblom/yellow-clarity/main/clarity-screenshot2.png" width="795" alt="Screenshot2"></p>
<p align="center"><img src="https://raw.githubusercontent.com/zenblom/yellow-clarity/main/clarity-screenshot3.png" width="380" alt="Screenshot3"><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><img src="https://raw.githubusercontent.com/zenblom/yellow-clarity/main/clarity-screenshot4.png" width="380" alt="Screenshot2"></p>

**Comment by gsegit** [comment, marked as answer]
Hello ;D

Very nice theme, thanks for sharing ;D
